docker/docker-compose.yaml

```diff
@@ -207,6 +207,7 @@ x-shared-env: &shared-api-worker-env
   WORKFLOW_CALL_MAX_DEPTH: ${WORKFLOW_MAX_EXECUTION_TIME:-5}
   SSRF_PROXY_HTTP_URL: ${SSRF_PROXY_HTTP_URL:-http://ssrf_proxy:3128}
   SSRF_PROXY_HTTPS_URL: ${SSRF_PROXY_HTTPS_URL:-http://ssrf_proxy:3128}
+  PROXY_FIX_MIDDLEWARE_ENABLED: ${PROXY_FIX_MIDDLEWARE_ENABLED:-false}
```

Comment:
If it's treated this config as an uncommon one for most users, we don't have to add it into the `env.example` and `docker-compose.yaml`. Just leave it as a inner config.

Reply:
Revert changes on .env.example and docker-compose.yaml as suggested,